dashbuilder-client/dashbuilder-dataset-editor/src/main/java/org/dashbuilder/dataset/editor/client/screens/DataSetDefWizardScreen.java

```diff
@@ -233,12 +223,10 @@ void onSave(final DataSetDef dataSetDef, final String message) {
         }
```

Comment:
I'm testing this manually and now after successfully creating SQL dataset I'm getting NPE on line 222 on line
`currentWorkflow.clear()`. Could you please revisit this fix?
![npe_afterdatasetcreationfinished](https://cloud.githubusercontent.com/assets/2716069/24288962/a0798fba-107f-11e7-9e71-33f858380762.png)